
bool PROTECTIVE_FIELD
bool PROTECTIVE_MODE
bool PROTECTIVE_FAULT
bool PROTECTIVE_OVERSPEED
bool PROTECTIVE_STABILITY
bool PROTECTIVE_SAFEGUARDED
